websites/D/Discord Türkiye/dist/metadata.json

```
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "can",
+		"id": "613700645173592086"
+	},
+  "contributors": [
+    {
+      "name": "nowongame",
+      "id": "686990422198845452"
+    }
+  ],
+	"service": "Discord Türkiye",
```

the website spells this differently
```suggestion
	"service": "Discord Turkiye",
```